src/resolver/resolve_path.zig

```diff
@@ -75,11 +75,13 @@ pub fn isParentOrEqual(parent_: []const u8, child: []const u8) ParentEqual {
         parent = parent[0 .. parent.len - 1];
```

Comment:
Should remove trailing slashes for `child` too, else `.equal` will be inaccurate